<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>415</width>
        <height>239</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QmyVideoWidget" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/001.GIF</normaloff>:/images/images/001.GIF</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPlay">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/620.bmp</normaloff>:/images/images/620.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPause">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/622.bmp</normaloff>:/images/images/622.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/624.bmp</normaloff>:/images/images/624.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSound">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/volumn.bmp</normaloff>:/images/images/volumn.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFullScreen">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/237.GIF</normaloff>:/images/images/237.GIF</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>无曲目</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderPosition">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>进度</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmyVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qmyvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
